File: src/Client/src/app/api/models/book-recommendation-for-meeting-dto.ts
/* tslint:disable */
/* eslint-disable */
import { BookDto } from '../models/book-dto';
import { UserSimpleDto } from '../models/user-simple-dto';
export interface BookRecommendationForMeetingDto {
  book: BookDto;
  recommendedBy: UserSimpleDto;
}

File: src/Server/WebApi/Dtos/BookRecommendationForMeetingDto.cs
namespace Cbc.WebApi.Dtos;

using Cbc.WebApi.Interfaces;
using Cbc.WebApi.Models.Entities;

public class BookRecommendationForMeetingDto : IMapFrom<BookRecommendation>
{
    public BookDto Book { get; set; } = null!;

    public UserSimpleDto RecommendedBy { get; set; } = null!;
}
